entries/QUnit.stack.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="QUnit.stack">
+	<title>QUnit.stack()</title>
+	<signature>
+		<argument name="offset" type="Number" optional="true" default="2">
+			<desc>Set the stackTrace line offset.</desc>
+		</argument>
+	</signature>
+	<desc>Return a single line source reference from the stacktrace (call stack)</desc>
+	<longdesc>
+		<p>This method returns a single line string representing the stacktrace from where it was called.</p>
+		<p>If it's given a offset argument, QUnit.stack will return the correspondent line from the call stack.</p>
+	</longdesc>
+	<example>
+		<desc>
+			The stack can be used on custom assertions and different reporters.
+		</desc>
+		<code><![CDATA[
+// QUnit log will be called to inform each assert execution
+QUnit.log( function() {
+	// offset == 7 will return the place where the assertion was called.
+	console.log( QUnit.stack( 7 ) );
+});
+
+QUnit.test( "foo", function( assert ) {
+	// the log callback will report the position of the following line.
```

Comment:
thanks. I'm don't check code style with the same accuracy when I'm writing documentation.